test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperOutputTest.cs

```diff
@@ -154,7 +115,7 @@ public void GenerateStartTag_ReturnsNothingIfWhitespaceTagName()
             var output = tagHelperOutput.GenerateStartTag();
 
             // Assert
-            Assert.Empty(output);
+            Assert.Null(output);
```

Comment:
It'd be a waste to create a `TagHelperContent` for this though, the start/end tag will never be huge strings which is the purpose behind the writer change in the first place. I see the consistency argument but the `GenerateContent` method will more often than not never be called by a user.

It's a throw up for sure but I'm leaning more towards leaving it as a string.